Incompatible version error on clutch-custom-gateway builds

Hey 👋 

**Description**
I'm following the `Development/Feature.Development` tutorial.
The 3 first parts works fine.
At the [4th](https://clutch.sh/docs/development/feature/#4-frontend-user-interface), I have an issue running the make.
If I use node version 16, the build fail asking for 18, and if I use 18, the build fail asking for 16.

**Expected Behavior**
The scaffolding shouldn't fail :-)

**Actual Behavior**
```bash
❯ nvm use v16
Now using node v16.14.0 (npm v8.3.1)
❯ make scaffold-workflow
cd tools/scaffolding && go run scaffolder.go -m frontend-plugin
2023/12/26 11:56:44 Welcome!
*** Analyzing environment...

*** Based on your environment, we've picked the following destination for your new workflow:
> ~/repositories/clutch/frontend/workflows
Is this okay? [Y/n]: n
Enter the destination folder [~/repositories/clutch/frontend/workflows]: ~/repositories/clutch-custom-gateway/frontend/workflows
Is this a wizard workflow? [Y/n]: Y
Enter the name of this workflow [Hello World]: Amiibo
Enter a description of the workflow [Greet the world]: Lookup all Amiibo by name
Enter the developer's name [asura]:
Enter the developer's email [tdeheurles@gmail.com]:

...

warning @clutch-sh/tools > react-scripts > workbox-webpack-plugin > workbox-build > @rollup/plugin-replace > magic-string > sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
[3/5] Fetching packages...
error react-hook-form@7.49.2: The engine "node" is incompatible with this module. Expected version ">=18". Got "16.14.0"
warning react-hook-form@7.49.2: The engine "pnpm" appears to be invalid.
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

2023/12/26 11:58:38 `yarn --frozen-lockfile` returned the above error
exit status 1
make: *** [Makefile:151: scaffold-workflow] Error 1

```
or
```
❯ nvm use v18
Now using node v18.6.0 (npm v8.13.2)
❯ make scaffold-workflow
cd tools/scaffolding && go run scaffolder.go -m frontend-plugin
2023/12/26 12:01:11 Welcome!
*** Analyzing environment...

*** Based on your environment, we've picked the following destination for your new workflow:
> ~/repositories/clutch/frontend/workflows
Is this okay? [Y/n]: n
Enter the destination folder [~/repositories/clutch/frontend/workflows]: ~/repositories/clutch-custom-gateway/frontend/workflows
Is this a wizard workflow? [Y/n]: Y
Enter the name of this workflow [Hello World]: Amiibo
Enter a description of the workflow [Greet the world]: Lookup all Amiibo by name
Enter the developer's name [asura]:
Enter the developer's email [tdeheurles@gmail.com]:

...

2023/12/26 12:02:17 cd /tmp/clutch-scaffolding-522436709 && /home/asura/repositories/Charlie/demo4/clutch/build/bin
/yarn.sh --frozen-lockfile && ~/repositories/clutch/build/bin/yarn.sh tsc &&  ~/repositories/clutch/build/bin/yarn.sh compile
yarn install v1.22.11
info No lockfile found.
[1/5] Validating package.json...
error @clutch-sh/amiibo@0.1.0: The engine "node" is incompatible with this module. Expected version ">=16.0.0 <17". Got "18.6.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

2023/12/26 12:02:17 `yarn --frozen-lockfile` returned the above error
exit status 1
make: *** [Makefile:151: scaffold-workflow] Error 1

```

**Version**
Clutch sha is a76909ab99e79280651081d92bf0282429629950

--- a/tools/scaffolding/templates/frontend/package.json
+++ b/tools/scaffolding/templates/frontend/package.json
@@ -32,7 +32,8 @@
     "@mui/material": "5.8.5",
     "@mui/lab": "5.0.0-alpha.87",
     "@mui/styles": "5.8.4",
-    "@mui/system": "5.8.5"
+    "@mui/system": "5.8.5",
+    "react-hook-form": "7.25.3"
   },
   "devDependencies": {
     "@clutch-sh/tools": "^2.0.0-beta"

